documentation/modules/deploying/proc-deploy-kafka-cluster.adoc

```diff
@@ -22,10 +22,14 @@ For a Kafka deployment, the following examples are provided:
 In this procedure, we use the examples for an _ephemeral_ and _persistent_ Kafka cluster deployment.
 
 Ephemeral cluster:: In general, an ephemeral (or temporary) Kafka cluster is suitable for development and testing purposes, not for production. This deployment uses `emptyDir` volumes for storing broker information (for ZooKeeper) and topics or partitions (for Kafka). Using an `emptyDir` volume means that its content is strictly related to the pod life cycle and is deleted when the pod goes down.
-Persistent cluster:: A persistent Kafka cluster uses `PersistentVolumes` to store ZooKeeper and Kafka data. The `PersistentVolume` is
-acquired using a `PersistentVolumeClaim` to make it independent of the actual type of the `PersistentVolume`. For example, it can use
-ifdef::Kubernetes[HostPath volumes on Minikube or]
-Amazon EBS volumes in Amazon AWS deployments without any changes in the YAML files. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.
+Persistent cluster:: A persistent Kafka cluster uses persistent volumes to store ZooKeeper and Kafka data. A `PersistentVolume` is acquired using a `PersistentVolumeClaim` to make it independent of the actual type of the `PersistentVolume`. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.
++
+A persistent Kafka cluster deployment can use the corresponding type of volume without any changes to the `Kafka` configuration:
+
+* Amazon EBS volumes on Amazon AWS
+* Azure Disk Storage on Microsoft Azure
+* Persistent Disk on Google Cloud
+* Local persistent volumes on bare metal
```

Comment:
This is IMHO confusing:
* These are only examples, so we need to make it clear that this is not limited to the 4 listed types.
* The _changes to the `Kafka` configuration_ could be also confusing. It can suggest you can change the type.

Maybe you can put it together as an example?
_If your Kubernetes cluster runs on AWS, Kubernetes might provision the Amazon EBS volumes. If you run on XXX, it might provision YYY._

Or rather remove it completely.